tests/tests/cpld_control_gfsv17

```diff
@@ -74,23 +75,60 @@ export_cpl
 export CPLCHM=.false.
 export DNATS=0
 
-export RESTART_N=3
+export RESTART_N=12
```

Comment:
Please do not change this, it is required for IAU test

Reply:
Thanks for pointing this out! Did not intentionally change this.  @mdtoyNOAA do you want to fix this or do you want me to make a PR to your branch?

Reply:
@JessicaMeixner-NOAA  If you don't mind, go ahead and make PR to my branch.

Reply:
Done https://github.com/mdtoyNOAA/ufs-weather-model/pull/5